uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/Hook/PageRun/CustomFieldIdsTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use CRM_Tasksassignments_Test_BaseHeadlessTest as BaseHeadlessTest;
+
+/**
+ * @group headless
+ */
+class CRM_Tasksassignments_Hook_PageRun_CustomFieldIdsTest extends BaseHeadlessTest {
+
+  public function testAddingTheCustomFieldIdsVarToAllowedPages() {
+    $resources = $this->prophesize('CRM_Core_Resources');
+    $hook = new CRM_Tasksassignments_Hook_PageRun_CustomFieldIds($resources->reveal());
+    $allowedPage = new Civi\Angular\Page\Main();
+
+    $hook->handle($allowedPage);
+
+    $resources->addVars('customFieldIds', [
```

Comment:
Shouldn't this prophecy be before the call to `$hook->handle()`?

Reply:
It works either way, but I'll move the prophecies to be before the `$hook->handle()` statement.

Reply:
Interesting, I didn't know that, always thought the prophecies had to be set up before the call, but maybe this is all checked after the test is run :man_shrugging: